client/src/connection_cache.rs

```diff
@@ -3,7 +3,9 @@ use {
         quic_client::QuicTpuConnection, tpu_connection::TpuConnection, udp_client::UdpTpuConnection,
     },
     lazy_static::lazy_static,
+    log::*,
```

Comment:
Where is this used?

Reply:
It's used by the inc_new_counter_info! macro.